doc/_static/mpl.css

```diff
@@ -1,3 +1,62 @@
+:root {
+  /* START COLOR PALETTE */
+
+  /* viridis solid colors */
+  --dark-purple: #440154;
+  --purple: #482475;
+  --blue: #355f8d;
+  --teal: #22a884;
+  --green: #7ad151;
+  --lime: #bddf26;
+
+  /* utility colors for text, links, etc... */
+  --grey: #4f4f4f;
+  --logo-blue: #11557c;
+  --pale-blue: #869fbb;
+  --pale-orange: #ffc9a5;
+  --orange: #ffaa70;
+  --bold-orange: #ffaa70;
+  --light-gray: #e5e5e5;
+  --medium-gray: #c4c4c4;
+  --black: #000;
+  --white: #fff;
+  --alpha-white: rgba(255, 255, 255, 0.8);
+  /* END COLOR PALETTE */
```

Comment:
Since these don't match pydata-sphinx-theme, I don't think we should add (most of ) them. The `--gray` and `--gray-dark` are too dark to use, and the greys in sidebar borders appear to be hard-coded, so I think we can hang on to `--light-gray`.

We can also keep the `--viridis` ones, though they should probably move to the theme eventually.

Reply:
I'll inline the ones I actually use.  @dorafc said exactly both of these things to me but I was lazy....